packages/next/src/server/image-optimizer.ts

```diff
@@ -534,7 +534,11 @@ export async function imageOptimizer(
   const { isAbsolute, href, width, mimeType, quality } = paramsResult
 
   if (isAbsolute) {
-    const upstreamRes = await fetch(href)
+    const upstreamRes = await fetch(href, {
+      headers: {
+        "Accept": "image/avif,image/webp,image/apng,image/svg+xml,image/*,*/*;q=0.8"
```

Comment:
`*/*` will match anything, right?

Is there an issue this is fixing? We need to add a test to confirm this fixes the issue.